packages/web/src/common/store/cache/tracks/sagas.ts

```diff
@@ -158,6 +164,61 @@ function* editTrackAsync(action: ReturnType<typeof trackActions.editTrack>) {
     }
   }
 
+  const getFeatureEnabled = yield* getContext('getFeatureEnabled')
+  const isEditTrackRedesignEnabled = yield* call(
+    getFeatureEnabled,
+    FeatureFlags.EDIT_TRACK_REDESIGN
+  )
+  if (isEditTrackRedesignEnabled && track.stems) {
+    const currentStemUploads = yield* select(getCurrentUploads, track.track_id)
```

Comment:
nit: maybe there's a better name that will help avoid the confusion we faced the first time we saw this. maybe like inProgressUploads or something?